torch/fx/graph.py

```diff
@@ -61,16 +62,19 @@ def map_arg(a, fn):
 
 class Graph:
```

Comment:
NB: you can declare

```
nodes: List[Node]
_used_names: Dict[str, int]
```

up outside of `__init__` which I think looks nice and is nicely documentary